src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -36,16 +39,75 @@ function getTypeOfCareById(id) {
 /**
  * Gets the type of visit that matches our array of visit constant
  *
- * @param {Object} id VAR appointment object
+ * @param {Object} id VAOS Service appointment object
  * @returns {String} type of visit string
  */
 function getTypeOfVisit(id) {
   return TYPE_OF_VISIT.find(type => type.id === id)?.name;
 }
 
+/**
+ * Finds the datetime of the appointment depending on the appointment type
+ * and returns it as a moment object
+ *
+ * @param {Object} appt VAOS Service appointment object
+ * @returns {Object} Returns appointment datetime as moment object
+ */
+function getMomentConfirmedDate(appt) {
+  if (appt.kind === 'cc' && appt.timeZone) {
+    const zoneSplit = appt.timeZone.split(' ');
+    const offset = zoneSplit.length > 1 ? zoneSplit[0] : '+0:00';
+    return moment
+      .utc(appt.appointmentTime, 'MM/DD/YYYY HH:mm:ss')
+      .utcOffset(offset);
+  }
+
+  const timezone = getTimezoneBySystemId(appt.locationId)?.timezone;
```

Comment:
Instead of using `appt.locationId` directly here, we'd want to use `appt.locationId?.substr(0, 3)`, so we only get the vista site.